lib/import-export-visitor.js

```diff
@@ -245,7 +257,22 @@ class ImportExportVisitor extends Visitor {
           dd.type = "ClassExpression";
         }
 
-        path.replace(buildExportDefaultStatement(this, dd));
+        // Almost every JS parser parses this expression the same way, but
+        // we should still give custom parsers a chance to parse it.
+        let ast = this.parse("module.exportDefault(ARG);");
```

Comment:
If you're giving custom parsers a chance should you then also be enforcing its structure with asserts below?